apis.md

```diff
@@ -9,3 +9,10 @@ The following OpenTracing APIs are currently available in the RFC stage:
 
 Please refer to the README files in the respective projects for examples of usage.
 
+### Implementation Notes
+
+OpenTracing APIs intend to express a similar experience to the user, while still being idiomatic to the language or framework. The following concepts apply to all variants.
+
+A **Span** is shared mutable state. It accumulates tags and logs, and can also create children. Finishing a span means capturing its state as a SpanRecord.
+
+A **SpanRecord** is an immutable representation of a completed span. It can be reported directly to an OpenTracing system, or encoded into a representation of an alternate system, such as HTrace or Zipkin.
```

Comment:
For the v1.0 API I'm less and less confident that we should really talk about the "standard tracer" at all here... it's only interesting to the tracing system implementors, and in terms of cardinality there are hopefully going to be far more "instrumentors" than implementors. Perhaps there should be a separate area of the docs for tracing system implementors, and that's where things like this could be discussed. Thoughts?